docs/howtos/DataPlane Generation - DPG/09tsp_client.mdx

```diff
@@ -0,0 +1,182 @@
+# Getting started with `tsp-client`
+
+`tsp-client` is a simple command line tool to facilitate generating client libraries from TypeSpec.
+
+## Installation
+
+```
+npm install -g @azure-tools/typespec-client-generator-cli
+```
+
+## Usage
+
+```
+tsp-client <command> [options]
+```
+
+## Commands
+
+Use one of the supported commands to get started generating clients from a TypeSpec project.
+This tool will default to using your current working directory to generate clients in and will
+use it to look for relevant configuration files. To specify a different output directory, use
+the `-o` or `--output-dir` option.
+
+### init
+
+Initialize the client library directory using a tspconfig.yaml. When running this command pass in a path to a local or remote tspconfig.yaml with the `-c` or `--tsp-config` flag.
+
+The `init` command generates a directory structure following the standard pattern used across Azure SDK language repositories, creates a [tsp-location.yaml](#tsp-locationyaml) file to control generation, and performs an initial generation of the client library. If you want to skip client library generation, then pass the `--skip-sync-and-generate` flag.
+
+> IMPORTANT: This command should be run from the root of the repository.
```

Comment:
can you give an example of the root of the repository? it can get confusing for monorepos, is it `azure-sdk-for-python` or is it `azure-sdk-for-python/sdk/containerregistry`

Reply:
oh i see you include it in the below example, if you feel like it you can expound on it more here, but def just a small itty bitty nit

Reply:
done, thanks for the suggestion!